i18next-parser.config.js

```diff
@@ -115,6 +75,12 @@ module.exports = {
   // Whether to use the keys as the default value; ex. "Hello": "Hello", "World": "World"
   // This option takes precedence over the `defaultValue` and `skipDefaultValues` options
 
-  verbose: true
+  verbose: true,
   // Display info about the parsing including some stat
+
+  i18nextOptions: {
```

Comment:
I wonder if this configuration should be when we use i18next in the backend/frontend instead, since here it only applies to the parser/task.

I'll do some tests later today.